test/btree2.c

```diff
@@ -9917,19 +9917,21 @@ main(void)
     unsigned         reopen;                    /* Whether to reopen B-tree during tests */
     const char      *driver_name;
     bool             api_ctx_pushed = false; /* Whether API context pushed */
+    int              localTestExpress;       /* localized TestExpress */
 
     driver_name = h5_get_test_driver_name();
 
     /* Reset library */
     h5_test_init();
-    fapl = h5_fileaccess();
+    fapl             = h5_fileaccess();
+    localTestExpress = TestExpress;
 
     /* For the Direct I/O driver, skip intensive tests due to poor performance */
-    if (!strcmp(driver_name, "direct"))
-        SetTestExpress(2);
+    if (localTestExpress < 2 && strcmp(driver_name, "direct"))
```

Comment:
As Larry mentioned in the meeting I think this should be !strcmp(...

Reply:
Fixed